packages/cli/src/utils/getWebpackConfig.ts

```diff
@@ -17,11 +17,19 @@ export type GetWebpackConfigOutput = {
   webpackConfig: webpack.Configuration;
 };
 
+async function buildProjectWebpackConfig(projectWebpackConfig) {
+  return typeof projectWebpackConfig === 'function'
+    ? projectWebpackConfig({ env: 'production' }, {}) // TODO read from args etc
```

Comment:
There is still a todo?

Reply:
its more a longterm one once we refactored the structure a bit. Webpack calls this fn with some processed args form the command line and we might need to fake all this